docs/pages/getting-started/docker-compose.mdx

```diff
@@ -0,0 +1,217 @@
+
+```bash
+User testuser has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h0m0s:
+https://localhost:3080/web/invite/your-token-here
+NOTE: Make sure localhost:3080 points at a Teleport proxy which users can access.
+```
+
+The Web UI will be available at the displayed URL. 
+
+(!docs/pages/includes/insecure-certificate.mdx!)
+
+Explore audit log, session recordings and other features of Teleport.
+
+## Next steps
+
+- Try out one of our [Helm Guides](../kubernetes-access/helm/guides.mdx).
+- Try out one of our [Database Access Guides](../database-access/guides.mdx).
+- Learn about [Teleport Server Access](../server-access/introduction.mdx).
+- Learn about [Teleport Server Access](../server-access/introduction.mdx).
+- Learn about [Teleport Access Control](../access-controls/getting-started.mdx).
+
+## Under the hood
+
+Let's unpack some of the setup that made the demo possible.
+Teleport's authentication is based on client certificates and certificate authorities.
+
+Here is `ssh.cfg` that instructs `ssh` client to use Teleport
+as a bastion server:
+
+```
+## Hosts with openssh suffix are OpenSSH nodes listening on port 22 as usual
+Host *.openssh.teleport
+    ProxyCommand ssh -o "ForwardAgent yes" -p 3023 proxy.luna.teleport -s proxy:%h:22
+
+# Hosts without openssh suffix are Teleport nodes listening on port 3022
+Host *.teleport !proxy.luna.teleport
+    ProxyCommand ssh -o "ForwardAgent yes" -p 3023 proxy.luna.teleport -s proxy:%h:3022
+```
+
+Ansible is set up to use ssh config above:
+
+```
+[defaults]
+host_key_checking = True
+inventory=/etc/teleport.d/ansible/hosts
+remote_tmp=/tmp
+
+[ssh_connection]
+scp_if_ssh = True
+ssh_args = -F /root/.ssh/config
+```
+
+OpenSSH server is set up to trust Teleport's CA and uses Teleport-issued
+host certificate:
+
+```
+TrustedUserCAKeys /mnt/shared/certs/teleport.pub
+HostKey /mnt/shared/certs/mars.openssh.teleport
+HostCertificate /mnt/shared/certs/mars.openssh.teleport-cert.pub
+```
+
+Teleport's user and role for used for bot access:
+
+```yaml
+kind: role
+version: v3
+metadata:
+  name: bot
+spec:
+  # SSH options used for user sessions 
+  options:
+    # max_session_ttl defines the TTL (time to live) of SSH certificates 
+    # issued to the users with this role.
+    max_session_ttl: 10h
+
+  # allow section declares a list of resource/verb combinations that are
+  # allowed for the users of this role. by default nothing is allowed.
+  allow:
+    logins: ['root']
+    node_labels:
+      '*': '*'
+---
+kind: user
+version: v2
+metadata:
+  name: bot
+spec:
+  roles: ['bot']
+```
+
+Our lab uses admin tool `tctl` to generate and export certs:
+
+```bash
+# Exports user CA for OpenSSH to trust.
+tctl auth export --type=user | sed s/cert-authority\ // > ./teleport.pub
+
+# Export host CA for SSH client to trust.
+tctl auth export --type=host > ./teleport-known_hosts.pub || return
```

Comment:
This needs some editing/tweaks to work out of the box as described in https://github.com/gravitational/teleport/pull/7480/files#r668825327

Reply:
Fixed for now with simple sed replace